packages/isomorphic/test/sha256.test.ts

```diff
@@ -0,0 +1,19 @@
+import { sha256 } from '@xrplf/isomorphic/sha256'
+import { bytesToHex } from '@xrplf/isomorphic/utils'
+
+describe('sha256', () => {
+  it('hashes', () => {
+    const h2a = sha256('abc')
+    const h2b = sha256
```

Comment:
Note: If you update the names of these variables in the examples in the README to communicate their purpose, please also update them in the test cases.